### PR TITLE
Add installation script for Windows in PowerShell.

### PR DESCRIPTION
**Pull request by moigagoo**

### CODE REVIEW

**Comment by moigagoo** [comment]
Quick note for Windows users. To build Nimph, you need to have cmake. The easiest way to install it on Windows (as well as other software) is through [scoop](https://scoop.sh):
```
scoop install cmake
```

**Comment by moigagoo** [comment]
Accidently closed and reopened.

File: bootstrap.ps1 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+if ( !(Join-Path 'src' 'nimph.nim' | Test-Path) ) {
```
Comment (moigagoo) [inline bootstrap.ps1]: I've ported the original script as is, but this line is weird. Why are we checking against src/nimph.nim when we really need to check whether the repos has been cloned yet? Shouldn't we check against nimph/src/nimph.nim?

File: bootstrap.ps1 [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  git clone git://github.com/disruptek/nimph.git
+  Set-Location nimph
+} 
+
+$env:NIMBLE_DIR = Join-Path $PWD 'deps'
```
Comment (moigagoo) [inline bootstrap.ps1]: Again, ported from the original, but bugs me. Installing nimph to a local dir doesn't make much sense—you want the tool to be available everywhere after the installation.

**Comment by disruptek** [comment]
We can change the test if it’s broken.  The two scenarios it’s designed to perform in:

- you cloned nimph and now you’re running the bootstrap from the current directory,
- you have only the bootstrap and you want to build nimph

There are two reasons it builds the binary but doesn’t install it anywhere:

1. I’m so tired of Nimble thinking it has rights to scribble in ~/.nimble that I’ve turned off permissions to that directory.  I don’t want nimph to irritate people the same way.
1. By letting the user install the app, it puts the responsibility of knowing where it is and how to invoke it on them, sidestepping documentation and confusion.

FWIW, some people are successful at simply installing nimph with nimble, but I think it’s unreliable with different versions of nimble and different environments and different nimterops, etc.  The bootstrap takes most of this out of the equation as it ensures that we get a fresh environment that should work consistently.

We can change whatever you want; this is just my rationale, and it may well be wrong!  😁

**Comment by disruptek** [comment]
Want to add the script, or mention of the script, to the README and maybe mention the bit about scoop/cmake, too?

**Pull request merged**